apps/fabric-website/index.html

```diff
@@ -53,6 +53,80 @@
         outline: none;
       }
     </style>
+    <script type="text/javascript">
```

Comment:
Adding this here is okay, but there are a couple other locations you'll need to add it to actually make it be used on the public site (this file is only used for the PR deploy of the website and a couple other secondary locations). 

The file that's actually used on the public site is in a private repo which I can point you to, but fabric-website/homepage.htm is a mirror of that file. So typically we check in updates to homepage.htm (and index.html) here first, then port them to the private repo.

Reply:
Completed .. will now add to the public-internal